karibu-dsl/src/main/kotlin/com/github/mvysny/karibudsl/v10/KCssVarNameA.kt

```diff
@@ -0,0 +1,10 @@
+package com.github.mvysny.karibudsl.v10
+
+/**
+ * style="line-height: var(--lumo-line-height-m);"
```

Comment:
I'm not sure what this means; could you please rewrite the kdoc, say: allows you to insert a CSS variable. Intended usage from [xyz] function. Example of the produced variable: xyz.